projects/go-json-iterator/build.sh

```diff
@@ -28,4 +28,4 @@ function compile_fuzzer {
   $CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer.a -lpthread -o $OUT/$fuzzer
 }
 
-compile_fuzzer fuzz Fuzz fuzz_json
+compile_fuzzer . Fuzz fuzz_json
```

Comment:
This is fixed from https://github.com/mdempsky/go114-fuzz-build/commit/df9b762fa80f6ffac8a321cc321c23830e6a2617